packages/strapi-helper-plugin/lib/src/components/HeaderNavWrapper/index.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+import colors from '../../assets/styles/colors';
+import sizes from '../../assets/styles/sizes';
```

Comment:
You need to use prop.theme in styled-components.

Reply:
This variables are not set in the Theme provider yet, the one currently set is only for the `strapi-admin` package so the variables are differents

Reply:
So you can add them to the theme provider

Reply:
It doesn't really concern this PR plus I prefer to wait for #5083 before doing so